frontend/src/components/Inquiries/AddInquiry.tsx

```diff
@@ -1,144 +1,153 @@
 import {
-    Button,
-    FormControl,
-    FormErrorMessage,
-    FormLabel,
-    Modal,
-    ModalBody,
-    ModalCloseButton,
-    ModalContent,
-    ModalFooter,
-    ModalHeader,
-    ModalOverlay, Textarea,
+  Button,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Textarea,
 } from "@chakra-ui/react"
-import {useMutation, useQueryClient} from "@tanstack/react-query"
-import {type SubmitHandler, useForm} from "react-hook-form"
+import { useMutation, useQueryClient } from "@tanstack/react-query"
```

Comment:
So these are all good whitespace changes, but remember how we talked about a PR should only have one purpose? If you want to fix up whitespace, I'd suggest doing it in a separate PR, to make this one easier to get through.

Reply:
@Eugene-Polonsky these are products of our different linters. we need a common config file

Reply:
we'll discuss